_IBC/crescent-evmos.json

```diff
@@ -2,22 +2,22 @@
     "$schema": "../ibc_data.schema.json",
     "chain_1": {
       "chain_name": "crescent",
-      "client_id": "07-tendermint-24",
-      "connection_id": "connection-27"
+      "client_id": "07-tendermint-12",
```

Comment:
why is this an edit to the crescent-stargaze file? Shouldn't this be a new file?

Reply:
Jejeje